build/onos-config-debug/Dockerfile

```diff
@@ -17,6 +17,9 @@ COPY --from=base /go/src/github.com/onosproject/onos-config/build/_output/onos-c
 COPY --from=base /go/src/github.com/onosproject/onos-config/build/_output/testdevice.so.1.0.0 /usr/local/lib/testdevice.so.1.0.0
```

Comment:
do you need the base ones here since this is the debug image?

Reply:
I thought it would be safer to copy them over in case there was an option of running **onit** in non-debug mode in future

Reply:
let's remove them for now. Also I think running `onit` in non debug mode would make use of `latest` for docker and not `debug`